### PR TITLE
DT-880 - Fix CSRA Bug

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/test/resources/net/syscon/elite/api/resource/impl/offender_extra_info.json [hunk 1]
```diff
@@ -97,13 +110,13 @@
       "assessmentSeq": 1
     }
   ],
```
Comment (Mjwillis) [inline src/test/resources/net/syscon/elite/api/resource/impl/offender_extra_info.json:112]: CSRA now removed

File: src/test/resources/net/syscon/elite/api/resource/impl/offender_extra_info_no_csra.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      "assessmentCode": "CATEGORY",
+      "assessmentDescription": "Categorisation",
+      "cellSharingAlertFlag": false,
+      "assessmentDate": "2019-01-01",
+      "nextReviewDate": "2019-10-22",
+      "assessmentStatus": "A",
+      "assessmentSeq": 1
+    }
+  ],
+  "category": "Cat B",
```
Comment (Mjwillis) [inline src/test/resources/net/syscon/elite/api/resource/impl/offender_extra_info_no_csra.json:42]: No CSRA when only a CAT assessment

File: src/test/resources/net/syscon/elite/api/resource/impl/offender_personal_care.json [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+      "assessmentCode": "CATEGORY",
+      "assessmentDescription": "Categorisation",
+      "cellSharingAlertFlag": false,
+      "assessmentDate": "2017-02-04",
+      "nextReviewDate": "2018-06-01",
+      "assessmentStatus": "A",
+      "assessmentSeq": 6
+    }
+  ],
+  "csra": "High",
```
Comment (Mjwillis) [inline src/test/resources/net/syscon/elite/api/resource/impl/offender_personal_care.json:144]: Both CSRA and CAT now shown

File: src/test/resources/net/syscon/elite/api/resource/impl/offender_personal_care.json [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    {
+      "type": "MERGED",
+      "value": "A3333AB",
+      "offenderNo": "A1234AA",
+      "bookingId": -1,
+      "issuedDate": "2017-08-28",
+      "caseloadType": "INST"
+    }
+  ],
+  "personalCareNeeds": [
```
Comment (Mjwillis) [inline src/test/resources/net/syscon/elite/api/resource/impl/offender_personal_care.json:161]: Added Personal Care Needs

File: src/main/java/net/syscon/elite/api/resource/impl/OffenderAssessmentResourceImpl.java [hunk 5]
```diff
@@ -48,13 +53,13 @@ public OffenderAssessmentResourceImpl(final InmateService inmateService) {
...
 
-        return inmateService.getInmatesAssessmentsByCode(offenderList, assessmentCode, latest, active);
+        return inmateService.getInmatesAssessmentsByCode(offenderList, assessmentCode, latest, active, false);
     }
 
     @Override
     public List<Assessment> postOffenderAssessmentsCsraList(final List<String> offenderList) {
         validateOffenderList(offenderList);
-        return inmateService.getInmatesAssessmentsByCode(offenderList, null, true, true);
+        return inmateService.getInmatesAssessmentsByCode(offenderList, null, true, true, true);
```
Comment (mikehalmamoj) [inline src/main/java/net/syscon/elite/api/resource/impl/OffenderAssessmentResourceImpl.java:62]: Maybe for a future PR, but too many booleans in this method signature = true?

**Pull request merged**